pom.xml

```diff
@@ -93,7 +93,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <external.build.directory>${project.basedir}/target</external.build.directory>
-    <beam.javadoc_opts />
+    <beam.javadoc_opts>-Xdoclint:all -Xdoclint:-missing</beam.javadoc_opts>
```

Comment:
Eventually, we want `missing` back, yes? Or I guess checkstyle also enforces our choice of where they should be?